README.md

```diff
@@ -222,6 +271,18 @@ General facultative options:
                         covered to consider a gene to be present (value
                         between [0, 100]). One INDEL will be considered as one
                         difference (default: 80)
+  --bowtieAlgo="--very-sensitive-local"
+                        Bowtie2 alignment mode. It can be an end-to-end
+                        alignment (unclipped alignment) or local alignment
+                        (soft clipped alignment). Also, can choose between
+                        fast or sensitive alignments. Please check Bowtie2
+                        manual for extra information:
+                        http://bowtie-bio.sourceforge.net/bowtie2/index.shtml .
+                        This option should be provided between quotes and
+                        starting with an empty space
+                        (like --bowtieAlgo " --very-fast") or using equal
+                        sign (like --bowtieAlgo="--very-fast")
+                        (default: "--very-sensitive-local")
```

Comment:
Default value does not start with an empty space

Reply:
The default only displays the default value that will be used